docs/host-powershell/sample-dotnet1.1/Logic/Logic.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
```

Comment:
Becuase `Logic.dll` is an assembly Library while `MyApp.exe` is an executable. If you inspect the dependencies of `Microsoft.NetCore.App`, you will see that it depends on `NetStandard.Library` and also include the runtime and host related packages.